CHANGELOG.md

```diff
@@ -2,6 +2,7 @@
 
 - [v1.3.0-alpha.3](#v130-alpha3)
   - [Downloads](#downloads)
+- [v1.3.0-alpha.3](#v130-alpha3)
```

Comment:
@david-mcmahon I'm seeing this happening in other PRs, such as https://github.com/kubernetes/kubernetes/pull/25524/files#r63061933. Are you planning to send a PR to fix this?